src/main/webapp/i18n/de/lti.json

```
@@ -13,9 +13,9 @@
                 }
             },
             "startExercise": "<strong>Herzlich Willkommen bei Artemis!</strong> Klicke auf den Button <i>Aufgabe starten</i>!",
+            "ltiSuccessLoginRequired": "LTI Übung erfolgreich gestartet. Bitte melde dich als Nutzer {{user}} an.",
```

```suggestion
            "ltiSuccessLoginRequired": "LTI Übung erfolgreich gestartet. Bitte melde dich als Nutzer:in {{user}} an.",
```
The [guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/) and our other translations use this form instead